stroom-core-client/src/main/java/stroom/data/client/presenter/SelectionSummaryPresenter.java

```diff
@@ -76,8 +77,12 @@ public void onHide(final boolean autoClose, final boolean ok) {
     private void update(final String postAction, final String action, final SelectionSummary result) {
         final SafeHtmlBuilder sb = new SafeHtmlBuilder();
```

Comment:
For info you can use TooltipUtil for this sort of HTML building.  Most of our multi-line tooltips are built with that. Feel free to add to it if there is anything extra you need it to do.

Reply:
@at055612 Thanks - will keep that in mind.